### ISSUE TITLE
Unhandled `InvalidStateError`

### ISSUE THREAD

**Issue by felipelube** [onfido/onfido-sdk-ui#2165]
### What was the expected behaviour?

No unhandled error.

### What happened instead?

We have seen some reports of the following `InvalidStateError`:

![image](https://github.com/onfido/onfido-sdk-ui/assets/77070869/b01189fd-ecc7-415c-8884-c3924b16ddbf)

```
InvalidStateError: Failed to execute 'createImageBitmap' on 'Window': The ImageBitmap could not be allocated.
```

### Version info:

- Onfido SDK: 12.2.1
- OS: Linux (Android 10)
- Browser:Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36
- Device: Android K (?)
- Node: 12.16
- Framework or relevant dependencies (if applicable)

### Integration configuration:

```
{
      useModal: false,
      token,
      onComplete: onfidoComplete,
      onError: onfidoError,
      steps: [
        {
          type: 'welcome',
          options: {
            title: ' ',
            descriptions: [
              language.STUDENT_IDENTIFICATION_SUBTITLE1,
              language.STUDENT_IDENTIFICATION_SUBTITLE2,
            ],
          },
        },
        {
          type: 'document',
          options,
        },
        {
          type: 'face',
          options: {
            requestedVariant: 'motion',
            motionFallbackVariant: 'standard',
          },
        },
        'complete',
      ],
    }
```

### Steps to reproduce:

From the sentry logs, seems the user is about to start the live video capture. 

### Reproducible demo:

Not able to provide at this time.

**Comment by warrenseine** [comment]
Thank you for the clear bug report. I understand that getting a reproducible case for this error might be complex.

I have managed to reproduce the error by forcing a delay in the webcam initialisation. We've never seen this error live ourselves so it is appreciated to see that your systems have logged it.

We are addressing the issue and will ship a fix in a next SDK release.

Nevertheless, I believe this error has no impact on the user flow as liveness processing will recover when the webcam properly initialises. It simply generates noise.

**Comment by felipelube** [comment]
Hey @warrenseine thanks! Knowing that this will not affect the user flow is reassuring, thanks for that. Let me know if I can help in any way forward.

**Comment by warrenseine** [comment]
The fix is available in 12.4.0, released today.
